services/workflows-service/src/document/document.controller.external.ts

```diff
@@ -108,6 +108,60 @@ export class DocumentControllerExternal {
     });
   }
 
+  @Get('tracker/:workflowId')
+  @ApiForbiddenResponse()
+  @HttpCode(200)
+  @ApiResponse({
+    status: 200,
+    description: 'Documents retrieved successfully',
+    schema: Type.Object({
+      business: Type.Array(Type.Record(Type.String(), Type.Any())),
+      individuals: Type.Object({
+        ubos: Type.Array(Type.Record(Type.String(), Type.Any())),
+        directors: Type.Array(Type.Record(Type.String(), Type.Any())),
+      }),
+    }),
+  })
+  @Validate({
+    request: [
+      {
+        type: 'param',
+        name: 'workflowId',
+        schema: Type.String(),
+      },
+    ],
+    response: Type.Any(),
+  })
+  async getDocumentsByWorkflowId(
+    @Param('workflowId') workflowId: string,
+    @CurrentProject() projectId: TProjectId,
+  ) {
+    return await this.documentService.getDocumentTrackerByWorkflowId(projectId, workflowId);
+  }
+
+  @Post('request')
```

Comment:
Thoughts? Or some similar name? 🙂 
```suggestion
  @Post('request-upload')
```